### PR TITLE
Added a caution note about not using PHP templates in Symfony

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | #5760 |

### CODE REVIEW

File: book/templating.rst [hunk 1]
```diff
@@ -51,6 +51,12 @@ template - a text file parsed by PHP that contains a mix of text and PHP code:
         </body>
     </html>
 
+.. caution::
+
+    The use of PHP templates is strongly discouraged for Symfony applications
+    because its support will be dropped in future Symfony versions. Create all
+    your templates using the Twig templating language.
```
Comment (xabbuh) [inline book/templating.rst]: To not confuse the reader I suggest to move this after the introduction of Twig.

**Comment by javiereguiluz** [comment]
I've reworded this note and I've moved to the "Why Twig?" sidebar.

**Comment by javiereguiluz** [comment]
Can this PR be considered finished or would you like any other change? Thanks.

**Comment by WouterJ** [comment]
Templating won't be dropped for at least 2 years now. I don't think it makes sense to add this to the documentation.

**Comment by xabbuh** [comment]
I agree with @WouterJ. We can discuss this again when a final decision is made for the core.

**Pull request closed without merging**